src/java/com/sarality/app/view/dialog/CustomViewDialogFragment.java

```diff
@@ -85,6 +85,8 @@ protected void reinitialize() {
    * @param loaderId Id to used for the loader
    */
   protected <D> void startLoad(DataSource<D> dataSource, int loaderId, DataConsumer<D> consumer) {
+    // Kill all previously loaded data by the activity
+    getActivity().getSupportLoaderManager().destroyLoader(loaderId);
```

Comment:
it is basically saying that the task is already loaded , i will just give you the results.
in my case, the datasource was null but i will still getting result from previous asyncTask in consume data